style.css

```diff
@@ -5,189 +5,205 @@
 }
 /*CSS RESET*/
 
+@font-face {
+  font-family: "AVGARDD";
+  src: url(/fonts/AVGARDD.TTF);
+}
+@font-face {
+  font-family: "AVGARDN";
+  src: url(/fonts/AVGARDN.TTF);
+}
+@font-face {
+  font-family: "impact_imp";
+  src: url(/fonts/impact.ttf);
+}
+body{
+  display:flex;
+  flex-direction: column;
+  background-color: #efefef
+}
 .header {
   height: 285px;
-  width: 100%;
   background-image: url(images/header-bg.jpg);
 }
-.header-content {
+.header__content {
   display: flex;
   justify-content: center;
   align-items: end;
 }
-.header-content span {
-  display: inline-block;
-}
 .logo {
+  position:relative;
+  left:-6px;
   background-image: url(images/logo.gif);
-  width: 372px;
+  background-repeat: no-repeat;
+  width:50%;
   height: 91px;
-  margin-left: 30px;
 }
-.pagetitle {
+.page__title {
   background-image: url(images/header-img1.jpg);
-  width: 311px;
+  background-repeat: no-repeat;
+  background-position: 90%;
+  width:50%;
   height: 215px;
 }
-.menubar {
+.menu__bar{
   display: flex;
-  height: 32px;
-  justify-content: center;
-  align-items: end;
-  position: absolute;
-  top: 215px;
-}
-.menubar ul {
-  list-style-type: none;
 }
-.menubar li {
-  float: left;
+.menu__items{
+  width:50%;
 }
 
-.btndivider {
-  vertical-align: middle;
-  background-image: url(images/menu-devider.gif);
-  background-repeat: no-repeat;
-  display: inline-block;
-  width: 2px;
-  height: 32px;
-}
-.hidden {
-  background-image: none;
-}
-.menubar a {
+.menu__items ul{
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width:360px;
```

Comment:
Fi consistent cand scrii. De exemplu mereu sa ai un spatiu intre doua puncte si stilul aplicat. Adica, in loc de `width:360px;`  scrie `width: 360px;`. La fel si pentru elemente de genul `.menu__items ul{` corect este `.menu__items ul {`. Ajuta foarte mult daca incepi sa scri consistent, incindental si in alte parti in care nu ai scris nicioadata. Disciplina asta iti va ramane si ti se va dezvolta usor usor atentia la detaliu. Angajatorii observa chestiile astea